README.md

```diff
@@ -157,76 +229,27 @@ Rosetta requires a few Celo Core Contracts
 
 * The list of required contracts is defined on `scripts/gen-contracts.go` file
 * Generation requires acces to `celo-blockchain` & `celo-monorepo`.
-* Generation assumes both projects are **already properly built**
+* Generation assumes both projects are **already properly built** (see above under instructions for running `rosetta` from source for more details on how to do this)
 * To run generator do `make gen-contracts`
 
-## How to build Docker Image
-
-Commands:
-  * `make docker-build`
-  * `make docker-publish`
-
 ## How to run rosetta-cli-checks
+
 Install the [`rosetta-cli`](https://github.com/coinbase/rosetta-cli) according to the instructions. Current testing has been done with `v0.5.16`.
```

Comment:
Make this a bullet point. I glossed over it as a to-do.

Reply:
✅